cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -99,23 +99,42 @@ load_etc_os_release() {
 configure_kernel_module_locking() {
   info "Checking if third party kernel modules can be installed"
   local -r kernel_cmdline="$(cat /proc/cmdline)"
-  # Assume that kernel commandline will never contain "lsm.module_locking=1",
-  # which is the default value when unspecified.
-  if echo "${kernel_cmdline}" | grep -q -v "lsm.module_locking=0"; then
-    local -r esp_partition="/dev/sda12"
-    local -r mount_path="/tmp/esp"
-    local -r grub_cfg="efi/boot/grub.cfg"
-
-    mkdir -p "${mount_path}"
-    mount "${esp_partition}" "${mount_path}"
-
-    pushd "${mount_path}"
+  local -r esp_partition="/dev/sda12"
+  local -r mount_path="/tmp/esp"
+  local -r grub_cfg="efi/boot/grub.cfg"
+  local need_reboot="false"
+
+  mkdir -p "${mount_path}"
+  mount "${esp_partition}" "${mount_path}"
+  pushd "${mount_path}"
+
+  # COS m77+ uses module signature to verify kernel modules.
+  if echo "${kernel_cmdline}" | grep -q "module.sig_enforce"; then
```

Comment:
Would it be clearer if we implemented this as something like:

if [[ "${VERSION}" -ge 77 ]]; then
else
fi

I think "${VERSION}" will have the milestone from sourcing ROOT_OS_RELEASE.